mst/page-index/page-index.js

```diff
@@ -38,21 +38,37 @@ module.exports = [
         title: "Viewing service logs",
         href: "viewing-service-logs",
       },
-      {
-        title: "Logging",
-        href: "logging",
-      },
       {
         title: "VPC peering",
         href: "vpc-peering",
       },
       {
-        title: "Send metrics to Datadog",
-        href: "metrics-datadog",
-      },
-      {
-        title: "Integrate Managed Service for TimescaleDB and Google Data Studio",
-        href: "google-data-studio-mst",
+        title: "Integrations",
+        href: "integrations",
+        excerpt: " MST integrates with the other tools you are already using, and makes it easy to add more integrations",
+        children:  [
+          {
+           title: "Integrate Managed Service for TimescaleDB and Google Data Studio",
```

Comment:
Let's change these titles to be parallel with each other. Might be best to use "verbing with product" as a structure, so we can have "Visualizing data with Grafana" and "Visualizing data with Google Data Studio" (or whatever).